tools/setup_helpers/BUILD.buck

```diff
@@ -0,0 +1,41 @@
+python_library(
```

Comment:
Is the plan to unify all of these BUILD.buck files with the internal build?

Reply:
Most of them will be unified/shared. I plan to avoid any internal changes right now so we can land it quickly and start getting OSS CI signals. Then gradually unify them.